pkg/crc/machine/vsock.go

```diff
@@ -146,3 +139,39 @@ func getSSHTunnelURI() string {
 	}
 	return u.String()
 }
+
+func HTTPAndHTTPS() []types.ExposeRequest {
+	hPort, hsPort := httpPort, httpsPort
+	if isPortInUse(hPort) {
+		logging.Warnf("Port %s is already in use. Going to use %s", httpPort, alternateHTTPPort)
```

Comment:
`Using port %s instead` might sound nicer.

Reply:
Is this the very last warning which is printed? Or is this going to be lost in the middle of the startup logs of crc?

Reply:
`Going to use` does not sound conclusive. `Assigned %s instead`, as `use` followed by `using` sounds repetitive.

Reply:
> Is this the very last warning which is printed? Or is this going to be lost in the middle of the startup logs of crc?

@cfergeau It is not the last warning, this is something part of info messages (Since it is warning so should've the `WARN` and different color)